superset/__init__.py

```diff
@@ -202,7 +201,6 @@ def index(self):
     base_template='superset/base.html',
     indexview=MyIndexView,
     security_manager_class=custom_sm,
-    update_perms=get_update_perms_flag(),
```

Comment:
I think we should make this `False` now, and assume people will have to run `superset init` once per deploy

Reply:
I think now we need to call `appbuilder.add_permissions(update_perms=True)` from `superset.cli.init`

Reply:
FAB defaults to `True` on `update_perms`, so that we get the same behavior.
My reasoning is that the present change will create all permissions on start just like before.
If we want to disable the permission creation (avoid contention on the db on startup), 
FAB_UPDATE_PERMS=False and run `superset init` once per deploy.

I would say that `appbuilder.add_permissions(update_perms=True)` can be achieved by calling the new `flask fab create-permissions`. I noticed that `superset db upgrade` creates and upgrades the db, leaving it already with bootstrap permissions and `superset init` created Superset's extra roles even with `FAB_UPDATE_PERMS=False` (so all is good).

Reply:
Ok, let's add another note in `UPDATE.md` about `FAB_UPDATE_PERMS = False` being the new equivalent to the envvar `SUPERSET_UPDATE_PERMS=0`.

I think it'd be good to call `appbuilder.add_permissions(update_perms=True)` within `superset init`, so that we people don't have to also `flask fab create-permissions` at every deploy. I don't think using `create-permissions` is documented anywhere on our side. I think it's easier to make `superset init` do more instead of documenting  using  `flask fab`

Reply:
Makes sense, new commit.